<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(193, 244, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4,1,0,1,1,1">
          <item>
           <widget class="QLabel" name="label">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>100</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/icon/iot_platform.png);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="topMargin">
             <number>10</number>
            </property>
            <property name="bottomMargin">
             <number>10</number>
            </property>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="styleSheet">
               <string notr="true">font: 20pt &quot;华文琥珀&quot;;
color: rgb(16, 90, 154);</string>
              </property>
              <property name="text">
               <string>物联网综合管理平台</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="styleSheet">
               <string notr="true">font: 75 14pt &quot;Times New Roman&quot;;
color: rgb(16, 90, 154);</string>
              </property>
              <property name="text">
               <string>IOT Integrated Management Platform</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QToolButton" name="device_monitor">
            <property name="styleSheet">
             <string notr="true">font: 10pt &quot;宋体&quot;;
color: rgb(16, 90, 154);
font: bold;</string>
            </property>
            <property name="text">
             <string>设备监测</string>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/icon/device_monitor.png</normaloff>:/icon/device_monitor.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="net_info">
            <property name="styleSheet">
             <string notr="true">font: 10pt &quot;宋体&quot;;
color: rgb(16, 90, 154);
font: bold;</string>
            </property>
            <property name="text">
             <string>网络信息</string>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/icon/net_info.png</normaloff>:/icon/net_info.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="parameter_set">
            <property name="styleSheet">
             <string notr="true">font: 10pt &quot;宋体&quot;;
color: rgb(16, 90, 154);
font: bold</string>
            </property>
            <property name="text">
             <string>参数配置</string>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/icon/parameter_set.png</normaloff>:/icon/parameter_set.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="system_set">
            <property name="styleSheet">
             <string notr="true">font: 10pt &quot;宋体&quot;;
color: rgb(16, 90, 154);
font: bold</string>
            </property>
            <property name="text">
             <string>系统设置</string>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/icon/system_set.png</normaloff>:/icon/system_set.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="restart">
            <property name="styleSheet">
             <string notr="true">font: 10pt &quot;宋体&quot;;
color: rgb(16, 90, 154);
font: bold</string>
            </property>
            <property name="text">
             <string>设备重启</string>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/icon/restart.png</normaloff>:/icon/restart.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QToolButton" name="Window_min">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/icon/window_min.png</normaloff>:/icon/window_min.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLabel" name="label_sysTime">
            <property name="styleSheet">
             <string notr="true">font: 10pt &quot;宋体&quot;;
color: rgb(16, 90, 154);
/*border:2px solid rgb(16, 90, 154);*/
font: bold
</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QToolButton" name="Window_close">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/icon/window_close.png</normaloff>:/icon/window_close.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,3">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QToolButton" name="configuration">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;宋体&quot;;
color: rgb(16, 90, 154);
font: bold;</string>
          </property>
          <property name="text">
           <string>通信组态</string>
          </property>
          <property name="icon">
           <iconset resource="images.qrc">
            <normaloff>:/icon/configuration.png</normaloff>:/icon/configuration.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="video_monitor">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;宋体&quot;;
color: rgb(16, 90, 154);
font: bold;</string>
          </property>
          <property name="text">
           <string>视频监控</string>
          </property>
          <property name="icon">
           <iconset resource="images.qrc">
            <normaloff>:/icon/video_monitor.png</normaloff>:/icon/video_monitor.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="data_query">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;宋体&quot;;
color: rgb(16, 90, 154);
font: bold</string>
          </property>
          <property name="text">
           <string>数据查询</string>
          </property>
          <property name="icon">
           <iconset resource="images.qrc">
            <normaloff>:/icon/data_query.png</normaloff>:/icon/data_query.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextUnderIcon</enum>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(16, 90, 154);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
